bin/parcel-server.ts

```diff
@@ -1,11 +1,14 @@
-import { fork } from 'effection';
+import { main, Operation } from 'effection';
```

Comment:
Instead of `fork` having a the effect of actually starting an Execution frame, it is an operation now and as such just returns an immutable `Operation` object. The new way to make a top-level entry point is via the `main` function.